### PR TITLE
CLI Signal 1.8 Improvements

### PR DESCRIPTION
**Pull request by malnick**
Non-blocker, expected GA.

https://mesosphere.atlassian.net/browse/DCOS-8489

Results:

```
analytics.track('installer', {
    'bootstrap_id': '0f11ac4a2b11f5601c6171c61746776a556fe9b2',
    'provider': 'onprem',
    'source': 'installer',
    'variant': 'enterprise',
    'errors': 6,
    'install_id': 'e458f0f7-14fa-4893-85b9-edba23229c5a',
    'customerKey': null,
    'install_method': 'cli',
    'action_name': 'preflight'
})```

### CODE REVIEW

**Comment by malnick** [comment]
CC @pyronicide

**Comment by malnick** [comment]
@spahl Ready for review.

File: ext/dcos-installer/dcos_installer/installer_analytics.py [hunk 1]
```diff
@@ -20,14 +20,14 @@ def send(self, action, install_method, num_errors):
         # We set customer key from config to avoid loading the config during class init
         customer_key = backend.get_config().get("customer_key", None)
 
-        analytics.track(user_id=customer_key, anonymous_id=self.uuid, event=action, properties={
+        analytics.track(user_id=customer_key, anonymous_id=self.uuid, event="installer", properties={
             "provider": "onprem",
             "source": "installer",
-            "variant": os.environ["BOOTSTRAP_VARIANT"],
+            "variant": 'open' if os.environ["BOOTSTRAP_VARIANT"] == '' else 'enterprise',
```
Comment (spahl) [inline ext/dcos-installer/dcos_installer/installer_analytics.py]: This is not ok. We can't have enterprise implementation bleed into the open project.

**Comment by malnick** [comment]
@spahl ready for review

**Comment by spahl** [comment]
:ship:

**Comment by cmaloney** [comment]
🚢

**Pull request merged**